ui/src/helpers/api.js

```diff
@@ -46,6 +46,99 @@ export default class Api {
         return response.data;
     }
 
+    //#region User api
+
+    getUsers = async(filter = "") =>{
+        const response = await this.client.get(`/user/listUsers?usernameFilter=${filter}`);
+        return response.data;
+    }
+
+    getUserDetails = async(user) => {
+        const response = await this.client.get(`/user/${user.userId}`);
+        return response.data;
+    }
+
+    updateUser = async(user) => {
+        const response = await this.client.put(`/user/${user.userId}`,user);
+        return response.data;
+    }
+
+    deleteUser = async(user) => {
```

Comment:
get userId

Reply:
you mean pass as params UserId?

Reply:
user you don't need the whole user object here